keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -362,6 +363,52 @@ We may come up with more validation rules during the implementation phase.
 The API comments to the Job status API fields will be updated to make the contract
 clear.
 
+We also plan to add a validation rule to make sure that the terminal conditions
+(`Failed` and `Complete`) are only added to Job when all pods are terminated.
+For that we plan to follow the approach described [below](#terminating-pods-and-terminal-job-conditions).
```

Comment:
Tentatively added to the validation seciton.
I think this seems like a useful extension of validation. It will fail fast in case some users want to write a controller which adds Failed or Complete directly, and it would prompt them to think about the termination of pods.